Umbraco-Cloud/Security/index.md

```diff
@@ -0,0 +1,46 @@
+# Security on Umbraco Cloud
+
+## HTTPS & Certificates
+
+All Umbraco Cloud websites use HTTPS by default. Both the default {projectName}.s1.umbraco.io and custom domains are protected by periodically renewed certificates from Let's Encrypt. This service is offered as part of Umbraco Cloud by [Umbraco LATCH](/Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md)
+
+### Custom Certificates
+
+Custom certificates can be used with all custom domains. Please refer to [Managing Custom Certificates documentation](Umbraco-Cloud/Set-Up/Manage-Hostnames/Security-Certificates/index.md)
+
+### TLS support
+
+As of April 2020, we've deprecated support for TLS1.0 & TLS 1.1. TLS1.2 is the only supported TLS protocol going forward.
+
+### TLS Ciphers support
+
+Umbraco Cloud Websites support the following TLS ciphers in this order:
+
+- TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+- TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+- TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA
+- TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA
+- TLS_DHE_RSA_WITH_AES_256_GCM_SHA384
+- TLS_DHE_RSA_WITH_AES_128_GCM_SHA256
+- TLS_DHE_RSA_WITH_AES_256_CBC_SHA
+- TLS_DHE_RSA_WITH_AES_128_CBC_SHA
+
+### TLS1.2 by default in external services
+
+In order to integrate older external applications accessing Umbraco Cloud Websites you might have to modify the TLS support in the .Net application.
```

Comment:
```suggestion
In order to integrate older external applications by accessing Umbraco Cloud Websites you might have to modify the TLS support in the .NET application.
```

Reply:
It's the external applications which are accessing Umbraco Cloud Websites. Adding "by" changes the meaning. Should we change this into something else then?

Reply:
Ah, I see 🤔
Perheps we could say `to` instead?
```suggestion
In order to integrate older external applications to access Umbraco Cloud Websites you might have to modify the TLS support in the .Net application.
```

That keeps the origin meaning, right?